app/models/edition.rb

```diff
@@ -192,6 +194,42 @@ def gone?
     end
   end
 
+  def to_h
```

Comment:
The changes in this class seem to introduce a bunch of presentation concerns into this model. 

I think it'd make sense that we have this logic in a presenter, what do you think? 

We also could do with adjusting `to_json` to not be using document fields and instead just be serializing the edition model.